test/unit/math/rev/fun/to_soa_sparse_matrix_test.cpp

```diff
@@ -0,0 +1,77 @@
+#include <iostream>
+#include <stan/math/prim/fun/Eigen.hpp>
+#include <stan/math/rev/fun/to_soa_sparse_matrix.hpp>
+#include <stan/math/rev/core.hpp>
+#include <test/unit/math/rev/util.hpp>
+#include <vector>
+/*
```

Comment:
These should either be uncommented (preferrably) or removed

Reply:
They just needed to be uncommented. Left over from debugging